Tote.Infrastructure/Repositories/SportType/SportTypeReadRepository.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Dapper;
+using System.Data;
+using Tote.Application.SportType.Common.Interfaces;
+
+namespace Tote.Infrastructure.Repositories.SportType
+{
+    internal class SportTypeReadRepository : ISportTypeReader
```

Comment:
The best practise in programming architecture world seal class witch not intended for inheritance. For example in Kotlin this feature used by default and they need to explicit make their classes NOT sealed. On the other hand sealed classes proceed by CLR in 45 times faster but it only in nanosecond that is why we can't mesure performance  increasing in small app.